src/processes.c

```diff
@@ -807,6 +811,25 @@ static void ps_submit_fork_rate (derive_t value)
 
 	plugin_dispatch_values(&vl);
 }
+
+static void ps_submit_syscalls (derive_t value)
```

Comment:
It would be simpler if you would add a `char const *type` argument to the existing `ps_submit_fork_date()` function (and find an appropriate name, maybe `ps_submit_type()`).